public/docs/dart/latest/guide/setup.jade

```diff
@@ -1,58 +1,166 @@
+:marked
+  ## Creating a project in WebStorm
+
+  Using WebStorm to create an app is simple, once you complete some one-time setup.
+
+  1. Launch WebStorm.
+  1. If you haven't already done so,
+  [configure Dart support in WebStorm](https://webdev.dartlang.org/tools/webstorm#configuring-dart-support).
+  1. Choose **Create New Project** from the welcome screen,
+  or **File > New > Project...** from the menu.  A dialog appears.
+  1. Select **Dart** from the list on the left.
+  1. Set the location and template:
+     1. In the **Location** input field, check that
+        the project folder is where you want it.
+     1. Also in the **Location** field,
+        change the name of the project from `untitled` to whatever you choose, such as `angular_quickstart`.
+    1. Make sure that **Generate sample content** is checked.
+    1. Select **Angular QuickStart Example** from the list.
+    <br>
+    The form should look similar to the following:
+    <br>
+    <img src="images/create-ng2-project.png" alt="A screenshot of the New Project dialog, with the specified selections" style="border:1px solid">
+  1. Click **Create**.
+
+  WebStorm takes several seconds to analyze the sources and do other housekeeping.
+  This only happens once. After that, you'll be able to use WebStorm for the usual IDE tasks,
+  including running the app.
+
+  For more information on using WebStorm, see
+  [Installing and Using WebStorm](https://webdev.dartlang.org/tools/webstorm).
 
-  :marked
-    These files are organized as follows:
 
-  .filetree
-    .file angular_quickstart
+a#cli
+:marked
+  ## Using a template from the command line
+  [Stagehand](http://stagehand.pub/) gives you
+  command-line access to the same templates that WebStorm uses.
+  Assuming the Dart SDK and
+  [pub cache `bin` directory](https://www.dartlang.org/tools/pub/cmd/pub-global#running-a-script-from-your-path)
+  are in your path, here's how you use Stagehand to create an Angular project.
+
+  1. Install or update stagehand:
+  <br>
+  `pub global activate stagehand`
+  1. Create a directory for your project:
+  <br>
+  `mkdir angular_quickstart; cd angular_quickstart`
+  1. Use Stagehand with the appropriate template to create a skeleton app:
+  <br>
+  `stagehand web-angular-quickstart`
+   <br>
+   **Note:** 
+   Examples in this guide and tutorial are based on the QuickStart example,
+   which is in the `web-angular-quickstart` template.
+   A more general-purpose template is `web-angular`.
+  1. Get the app's dependencies: <br>
+     `pub get`
+  1. Run the app: <br>
+     `pub serve`
+
+
+.l-main-section#seed
+:marked
+  ## What's in the QuickStart example?
+
+    The <live-example name="quickstart">QuickStart example</live-example>
+    contains the following core files:
+
++makeTabs(`
+    quickstart/ts/app/app.component.ts,
+    quickstart/ts/app/main.ts,
+    quickstart/ts/index.html,
+    quickstart/ts/styles.css,
+    quickstart/dart/pubspec.yaml`,
+  ',,,,quickstart',
```

Comment:
Since you switched the order of the files, the region list needs to be adjusted as well. Use
```
',,,quickstart,',
```